packages/paste-core/components/breadcrumb/__tests__/breadcrumb.spec.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import {axe} from 'jest-axe';
+import {matchers} from 'jest-emotion';
+import {render, screen} from '@testing-library/react';
+import {Breadcrumb, BreadcrumbItem} from '../src';
+
+expect.extend(matchers);
```

Comment:
What's this for?

Reply:
That extends `expect` with `toHaveStyleRule`, so we can test for a specific style. I ended up using it a lot with Table, since thats just a bunch of styled Boxes. I'm using it here to test the `inline-flex` style.

Reply:
Might be worth looking if you can add this extend in the jest config so we have it everywhere